ui/packages/consul-ui/app/templates/application.hbs

```diff
@@ -85,20 +85,21 @@
               (hash Name=(env 'CONSUL_DATACENTER_LOCAL'))
             )
             dcs.data
-            as |dc dcs|
+          as |dc dcs|
           }}
             {{#if (and (gt dc.Name.length 0) dcs)}}
 
-              {{! figure out our current DC and convert it to a model }}
+            {{! figure out our current DC and convert it to a model }}
               <DataSource
                 @src={{uri
-                  '/${partition}/*/${dc}/datacenter-cache/${name}'
-                  (hash dc=dc.Name partition=partition name=dc.Name)
-                }}
+                '/${partition}/*/${dc}/datacenter-cache/${name}'
+                (hash dc=dc.Name partition=partition name=dc.Name)
+              }}
                 as |dc|
               >
                 {{#if dc.data}}
-                  <HashicorpConsul
+                  <DataSource @src={{uri '/${partition}/*/${dc}/hcp-link' (hash dc=dc.Name partition=partition name=dc.Name) }} @open={{true}} as |hcpLink|>
```

Comment:
Bit curious about this - Does this mean we'll fetch this upon application load? And this data will be accessible as the `hcpLink` variable in this template?

Would we still need/want this if we're only requesting this inside of the `hcpLinkStatus` service?

Reply:
good question. when application loading, we calling the service. but we have to call it on login as well
due to `<HashicorpConsul>` component has the tag structure and we need the data in sidenav and in main tags, I've put the call to application and make it `open`. it means that on login we will have it reloaded, same as on route change. 

It is questionable though, when we need to reload it. let's say if we are linking in HCP, and does not reload page in core, I think we still should get the information if it is linked

actually it depends on the cost of calling the service, which in our case seems to be not too bad, because it uses local consul... what do you think?